### PR TITLE
Closes #11 Add pytest and travis

### LINKED ISSUE #11: Add CI unittests with pytest and Travis
Add CI with https://travis-ci.org/

### PR DESCRIPTION
**Pull request by lgarest**
- Add source test venv and pytest in Makefile
- Update gitignore to ignore sensitive credentials file
- Need to integrate the app in it's own Travis namespace, so we can CI the repo and add the badge in the README file

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+language: python
+python:
+  - "3.6"
+
+# command to install dependencies
+install:
+  - pip install -r requirements/test.txt
```
Comment (semicolom) [inline .travis.yml]: `make test` command already does this

File: Makefile [hunk 2]
```diff
@@ -49,6 +49,10 @@ lint: virtualenv_test
 	$(FLAKE8) cunao.py
 
 test: virtualenv_test isort-check lint
+	( \
+	source .venv_test/bin/activate; \
+	pytest test/; \
+	)
```
Comment (semicolom) [inline Makefile]: You should replace this for: `$(PYTEST) tests/`
Define `PYTEST` in the Makefile like `FLAKE8` for example

File: test/test_sample.py [hunk 3]
```diff
@@ -0,0 +1,6 @@
+
```
Comment (semicolom) [inline test/test_sample.py]: Name the the folder `tests` instead of `test`

File: test/test_sample.py [hunk 4]
```diff
@@ -0,0 +1,6 @@
...
+
+# Just asserts that this file exists
```
Comment (semicolom) [inline test/test_sample.py]: Put the comments inside the function

**Comment by lgarest** [comment]
CI works as seen in https://travis-ci.org/pablobuenaposada/cunadobot/jobs/436518330

**Comment by lgarest** [comment]
Once it's merged, we can add the badge #18

**Pull request merged**